src/geo/projection/albers.js

```diff
@@ -41,33 +43,43 @@ function unproject(x, y, constants) {
     const y__ = y - 0.5;
     const x_ = (x__ - 0.5) * 2;
     const y_ = (y__ - 0.5) * -2;
-    const y_2 = -(y_ - b);
-    const theta = Math.atan2(x_, y_2);
-    const lng = (theta /n * 180 / Math.PI) + constants.refLng;
+    const y2 = -(y_ - b);
+    const theta = Math.atan2(x_, y2);
+    const lng = (theta / n * 180 / Math.PI) + constants.refLng;
     const a = x_ / Math.sin(theta);
     const lat = Math.asin((Math.pow(a / r * n, 2) - c) / (-2 * n)) * 180 / Math.PI;
 
     return new LngLat(lng, lat);
 }
 
+function albersProject(lng, lat) {
+    return project(lng, lat, albersConstants);
+}
+
+function alaskaProject(lng, lat) {
+    return project(lng, lat, alaskaConstants);
+}
+
 export const albers = {
     name: 'albers',
     range: [3.5, 7],
-    project: (lng, lat) => {
-        return project(lng, lat, albersConstants);
+    project: (lng: number, lat: number) => {
+        return albersProject(lng, lat);
     },
-    unproject: (x, y) => {
+    unproject: (x: number, y: number) => {
         return unproject(x, y, albersConstants);
-    }
+    },
+    tileTransform: makeTileTransform(albersProject)
 };
 
 export const alaska = {
     name: 'alaska',
     range: [4, 7],
-    project: (lng, lat) => {
-        return project(lng, lat, alaskaConstants);
+    project: (lng: number, lat: number) => {
+        return alaskaProject(lng, lat);
     },
```

Comment:
Redundant wrapper too. I guess this was done to make Flow happy? What if we type the functions above?